src/containers/login/Login.tsx

```diff
@@ -54,17 +59,47 @@ const Footer: typeof Paragraph = styled(Paragraph)<ParagraphProps>`
   margin-bottom: 10px;
 `;
 
+const LoginAlert = styled(Alert)`
+  width: 90%;
+  margin-top: -60px;
+  margin-bottom: 20px;
+`;
+
+const MobileLoginAlert = styled(Alert)`
+  width: 90%;
+  margin-bottom: 20px;
+`;
+
+const loginErrorMessage = 'The username or email you entered was incorrect.';
+const greetingHeader = 'Welcome Back!';
+const greetingBody =
+  'Dreamcatcher kogi taiyaki keytar. Swag typewriter craft beer\n' +
+  'cronut pok pok gentrify flannel salvia deep v pork belly\n' +
+  'pitchfork. Swag fashion axe fam. Occupy biodiesel jean shorts\n' +
+  'affogato PBR&B freegan bushwick vegan four loko pickled.';
+
 type LoginProps = UserAuthenticationReducerState;
 
 const Login: React.FC<LoginProps> = ({ tokens }) => {
   const { windowType, width } = useWindowDimensions();
   const dispatch = useDispatch();
-  const onFinish = (values: any) => {
+  const history = useHistory();
+  const privilegeLevel = useSelector((state: C4CState) =>
+    getPrivilegeLevel(tokens),
+  );
+  const [loginFailed, setLoginFailed] = useState(false);
+
+  const isLoggedIn: boolean =
+    privilegeLevel === PrivilegeLevel.STANDARD ||
+    privilegeLevel === PrivilegeLevel.ADMIN;
+
+  const onFinish = (values: LoginRequest) => {
     dispatch(login({ email: values.email, password: values.password }));
+    isLoggedIn
+      ? history.push('/home')
+      : setTimeout(() => setLoginFailed(true), 100);
   };
```

Comment:
```suggestion    
  if (privilegeLevel !== NONE) {
    history.push('/home')
  }

  const onFinish = (values: LoginRequest) => {
    dispatch(login({ email: values.email, password: values.password }));
  };
```

Essentially, this component will always be checking the global state to determine if the user is signed in. Whenever the user's privilege level is not NONE, we should immediately redirect them away from login.

It may seem odd to be checking privilegeLevel before we make the request, to be doing those independently, but this is because we are relying upon global state.

Reply:
Another side note: would be incredible if you could make a routes enum for `/home` and replace occurrences of each, that way we can alter routes with greater ease.